oauth2_http/java/com/google/auth/oauth2/GoogleCredentials.java

```diff
@@ -179,6 +184,22 @@ public static GoogleCredentials fromStream(InputStream credentialsStream,
   }
 
   /**
+   * Saves the end user credentials in specified file
+   *
+   * @param credentials InputStream containing user credentials in JSON format
+   * @param filePath Path to file where to store the credentials
+   * @throws IOException An error saving the credentials.
+   */
+  public void saveCredentialsToFile(InputStream credentials, String filePath) throws IOException {
```

Comment:
There is still nothing specific to credentials in this function. I would suggest moving it to `OAuth2Utils` so that it doesn't form part of the surface of the library (since `OAuth2Utils` is package-private). I suggest calling it "writeInputStreamToFile".